### DISCUSSION TITLE
Admin privileges gone from Netbox administration web page....

### DISCUSSION THREAD

**Discussion by ZachSwartz** [netbox-community/netbox#15003] [Q&A] [answered]
Today I updated our Netbox software from v3.3.3 to v3.7.1. After the update, I had a coworker message me and say he was having a permissions issue. I logged into the UI and went to the admin page, but it looks like my privileges have disappeared. The only options showing up are the following:

Python Social Auth
-Associations
-Nonces
-User Social Auths

System
-Background Tasks
-Installed Plugins

Then there is a list of users to the right that I can not click on or edit in any way. 

Any ideas what's going on here? Any help would be appreciated.

**Comment by ZachSwartz** [comment, marked as answer]
Nevermind, I'm and idiot....

**Reply by DanSheps** [reply]
It is all good, sometimes you just miss things

**Reply by jeremystretch** [reply]
The good news it that it's not a problem with your permissions! :slightly_smiling_face:

**Reply by fouram** [reply]
~~Okay, I'm still an idiot. Same thing here except I see Recent Actions. What am I missing?~~

Wow yikes, I'm going to bed...

(All the Netbox functionality like user management and custom fields have been moved to the new bottom three options on the sidebar, provided you have an admin-privileged account)
